ts/Series/Bubble/BubbleSeries.ts

```diff
@@ -737,6 +740,13 @@ class BubbleSeries extends ScatterSeries {
             const point = data[i];
             const radius = radii ? radii[i] : 0; // #1737
 
+            // Negative points means negative z values (#9728)
+            point.negative = (point as any)[zoneAxis] < (
+                (options as any)[zoneAxis + 'Threshold'] ||
+                threshold ||
+                0
+            );
```

Comment:
Can we take this opportunity to do this block typesafe? We learned that the zone axis check is only for the case of `z`, not for `x`. Besides, we already have `point.negative` computed for zoneAxis = 'y'. So I believe we can do something like this:

```ts
if (zoneAxis === 'z') {
    point.negative = (point.z || 0) < (options.zThreshold || 0);
}
```

Reply:
Ahh, ok! I didn't get it the first time. Will change it - thank you!